docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+
+  postgres:
+    image: postgres:14
+    environment:
+      POSTGRES_DB: adp-dev
+      POSTGRES_USER: adp
+      POSTGRES_PASSWORD: secret
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
+    ports:
+      - 127.0.0.1:5432:5432
+
+  adminer:
+    image: adminer
+    ports:
+      - 0.0.0.0:9000:8080
```

Maybe change that to

```suggestion
      - 127.0.0.1:9000:8080
```

in order to not expose that to the outside network?